climada/engine/impact.py

```diff
@@ -432,6 +432,10 @@ def plot_basemap_impact_exposure(self, event_id=1, mask=None, ignore_zero=True,
                          'instance with parameter save_mat=True')
             return []
 
+        if event_id not in self.event_id:
+            LOGGER.error('Event ID %s not found', str(event_id))
```

Comment:
Do you want to raise an error for the user? Then do so properly. The `LOGGER.error` is mostly for debugging purposes. In this case this might be the wanted behaviour.

Reply:
good spot